assets/js/state/selectors/index.js

```diff
@@ -91,3 +91,16 @@ export const getInstancesOnHost = (hostId) => (state) => {
 
   return [...foundApplicationInstances, ...foundDatabaseInstances];
 };
+
+export const getAllInstances = () => (state) => {
```

Comment:
I know that I'm coming again with this, but maybe `getAllSAPInstances` would be more meaningful.
Wdyt?